packages/core/contracts/financial-templates/contract-for-difference/ContractForDiffrenceCreator.sol

```diff
@@ -14,34 +14,34 @@ import "./ContractForDifference.sol";
 import "../common/financial-product-libraries/contract-for-difference-libraries/ContractForDifferenceFinancialProductLibrary.sol";
 
 /**
- * @title Contract For Difference Contract creator.
+ * @title Contract For Difference Contract Creator.
  * @notice Factory contract to create and register new instances of contract for difference contracts.
- * Responsible for constraining the parameters used to construct a new CFD. These constraints can evolve over time and are
- * initially constrained to conservative values in this first iteration.
+ * Responsible for constraining the parameters used to construct a new CFD. These constraints can evolve over time and
+ * are initially constrained to conservative values in this first iteration.
  */
 contract ContractForDifferenceCreator is Testable, Lockable {
     using FixedPoint for FixedPoint.Unsigned;
 
     // Address of TokenFactory used to create a new synthetic token.
-    address public tokenFactoryAddress;
+    TokenFactory public tokenFactory;
```

Comment:
Refactored this to rather store instances of the contract types, rather than just the addresses. This way we can pass in these types, like we do on the CFD contract and enabling us to pass around contract instances.